script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1133,17 +1142,17 @@ That was the final ray of light.
 [sha:4ba21fc23cdbccaeb34a528c77028a48195ef585]{
 -- Page 645, Offset 35122.
 -- たすかること。いきてでられることなんて、もう、だれもほしくはありませんでした。はやくたべられておわりたかったです。でも、彼の目は、わたしたちとはちがっていて。
-A plan to save me. It was too late for anyone to want to be part of a plan to let me out alive. They only wanted to be eaten quickly and get it over with. But the look in his eyes were different from any of us.
+He's going to save me. Everyone here has already given up on getting out of here alive. We all just want to be eaten already so that it'll all be over, but he's different. He has a different look in his eyes than any of us.
 }
 [sha:a093e39eb95381ef474a87c92418eaad0affae67]{
 -- Page 645, Offset 35123.
 -- そのこういは、だれかをたすけようと願うのは、とてもニンゲンらしいと、おもったのです。
-His kindness, and his wish to save others, felt extremely human.
+His kindness and his desire to help others feels so human.
 }
 [sha:af86d64165ba8b923ea970eaa0c67b7aef3306af]{
 -- Page 646, Offset 35124.
 -- わたしたちはつみあげました。肉製のスーツをつみあげます。それはてんごくへのかいだんみたいに、ぷーるのうエまでとどきました。
-We're all piled up on top of each other. Piled up like a suit made of flesh. Like a stairway to heaven, it reaches all the way to the top of the pool.
+We piled them up. We piled the suits made out of flesh up. Like a staircase to heaven, it extended all the way to the top of the pool.
```

Comment:
"stairway to heaven" is right even though it sounds like we're just referencing the led zeppelin song